spec/core/cache_spec.rb

```diff
@@ -101,7 +173,7 @@ def initialize(opts = {})
 
     id = SecureRandom.hex
 
-    # security groups on aws for eg can have the same identity group name 'default'.
+    # security gruops on aws for eg can have the same identity group name 'default'.
```

Comment:
I think I fixed this typo in my master also, looks like that change also was dropped.